src/app/src/main/kotlin/org/imperial/mrc/hint/security/tokens/GZipHelpers.kt

```diff
@@ -0,0 +1,73 @@
+package org.imperial.mrc.hint.security.tokens
+
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import java.io.InputStreamReader
+import java.util.*
+import java.util.zip.GZIPInputStream
+import java.util.zip.GZIPOutputStream
+
+fun String.inflated() = inflate(this)
+fun String.deflated() = deflate(this)
+
+private fun deflate(uncompressed: String): String
```

Comment:
I don't think we need any of this deflating/inflating business. This was added in Montagu because test users who had access to all modelling groups ended up with tokens that exceeded maximum HTTP request header sizes. We aren't encoding roles/permissions here so tokens will be reasonably short.